package.json

```diff
@@ -9,6 +9,8 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@aws-amplify/ui-react": "^5.3.1",
```

Comment:
I believe the Amplify UI library just adds premade UI components that are already set up to interface with their services. I omitted this since we have our own UIs set up already.

We haven't used them yet, so maybe just remove it for now? Thoughts @jackfales?